### DISCUSSION TITLE
remove background color of this window

### DISCUSSION THREAD

**Discussion by fallen9999** [DSheirer/sdrtrunk#1929] [General]
good evening could someone point me to the file that i can remove the background of the window that shows what's playing if you see the youtube video I'm trying to make it transparent 
![Screenshot 2024-05-30 181457](https://github.com/DSheirer/sdrtrunk/assets/647812/2189b7cd-573d-40b4-8602-45e4067d52bb)

https://www.youtube.com/watch?v=ZttaRyWLHfA

**Comment by fallen9999** [comment]
Please. i will donate :)

**Comment by TeddJohnson** [comment]
"Live Stream Recording not available"

you might be better served if you take screenshots and attach them directly
